backpack/custom_module/graph_utils.py

```diff
@@ -196,11 +238,161 @@ def _transform_flatten_to_module(module: Module, debug: bool) -> GraphModule:
     graph.lint()
 
     if debug:
-        print(f"\tFlatten transformed: {len(nodes)}")
+        print(f"\tFlatten functions transformed: {len(nodes)}")
+
+    return GraphModule(module, graph)
+
+
+def _transform_get_item_to_module(module: Module, debug: bool) -> GraphModule:
+    target = "<built-in function getitem>"
+    if debug:
+        print(f"\tBegin transformation: {target} -> ReduceTuple")
+    counter: int = 0
+    graph: Graph = BackpackTracer().trace(module)
+
+    for node in graph.nodes:
+        if node.op == "call_function" and target in str(node.target):
+            _change_node_to_module(
+                node,
+                "reduce_tuple",
+                module,
+                ReduceTuple(index=node.args[1]),
+                (node.args[0],),
+            )
+            counter += 1
```

Comment:
rewrite, create list of nodes first, then iterate over them -> less indented code